Bookish/Views/Catalog/BookList.cshtml

```diff
@@ -32,26 +58,98 @@
         </tr>
         </thead>
         <tbody>
-        @foreach (Book b in Model.CatalogEntries)
+        @for (int i = 0; i < Model.CatalogEntries.Count; i++)

```

Comment:
Why not use something like `foreach (var book in Model.CatalogEntries)`?

Reply:
Fixed 👍 it was originally a foreach but changed it to a for loop during an attempt to fix the Post/Get issue yesterday :(